share/util/shr_mct_mod.F90

```diff
@@ -828,7 +828,7 @@ subroutine shr_mct_sMatWritednc(sMat,iosystem, io_type, io_format, fileName,comp
    count(:) = -999
    pe_loc(:) = -999
    ssize(:) = 1
-   call MPI_GATHER(lsize,1,MPI_INTEGER,count,ssize,MPI_INTEGER,0,mpicom,ierr)
```

Comment:
I was tempted to rm ssize altogether from this routine, but I am not sure which lib calls it.

Reply:
I don't think this routine is called anywhere.  Looks like something added to help with debugging.